MaraBot/Core/PresetValidation.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+
+namespace MaraBot.Core
+{
+    /// <summary>
+    /// Holds validation logic for presets.
+    /// </summary>
+    public static class PresetValidation
+    {
+        /// <summary>
+        /// Randomizer version the validation is written for.
+        /// </summary>
+        public static readonly string VERSION = "1.22";
+
+        /// <summary>
+        /// List of all options.
+        /// </summary>
+        public static Dictionary<string, Option> AllOptions;
```

Comment:
Could we have this as a parameter in `ValidateOptions` instead? I think this would be cleaner to keep a static utility class such as `PresetValidation` with no static properties. Also, it should be easy to provide the `Options` getter to the commands that require it (`CustomRaceCommandModule` already has it).